packages/ckeditor5-page-break/src/pagebreakcommand.ts

```diff
@@ -63,7 +62,7 @@ function isPageBreakAllowedInParent( selection: DocumentSelection, schema: Schem
  * Returns a node that will be used to insert a page break with `model.insertContent` to check if the page break can be placed there.
  */
 function getInsertPageBreakParent( selection: DocumentSelection, model: Model ): Element {
-	const insertionRange = findOptimalInsertionRange( selection, model );
+	const insertionRange = model.schema.findOptimalInsertionRange( selection, model );
```

Comment:
This was using the widget implementation of this method. I think this should not change.

Reply:
Adressed in 24ac1fa